### ISSUE TITLE
Hub persists in 'ContainerCreating' state

### ISSUE THREAD

**Issue by jgerardsimcock** [jupyterhub/zero-to-jupyterhub-k8s#757]
I am running the the jupyterhub version-`v0.7-4c22c3d` on GKE with tiller version `2.9.1` and GKE `1.9.7-gke.3`. I can run helm install but the hub persists in `ContainerCreating` state.

**Comment by betatim** [comment]
How did you solve this? I've observed this behaviour sometimes with a recent z2jh deploy and we saw it at the weekend for the mybinder.org deployment as well.

**Comment by consideRatio** [comment]
Perhaps it is because of the introduction of new labeling and a hub-deployment and the associated selector labels being explicitly set as required by the k8s Deployment API v1 in k8s 1.9.

Personal TODO:
- [ ] Make a 0.6-0.7-dev upgrade with k8s 1
9 and helm 2.9.1 and see what happens.

**Comment by jgerardsimcock** [comment]
I frequently have to delete the old hub for the `helm upgrade` to be initialized. Sometimes, the hub is created anew with the helm upgrade command and sometimes, it seems, this is not the case. 

The error, in this particular instance, as far as I can tell, was that I had introduced a spelling error into the path for my jupyterhub custom templates. Once I noticed and corrected, the hub began working as intended.
